pytorch_lightning/callbacks/progress.py

```diff
@@ -27,7 +27,7 @@ def __init__(self):
                 self.enabled = True
 
             def disable(self):
-                self.enabled = False
+                self.enable = False
```

Comment:
@Borda but it also needs to be changed above. I will change it. please don't merge yet

Reply:
sure, sorry, just chack it with master code... and missed the example above